packages/smooth_app/lib/pages/product/new_product_page.dart

```diff
@@ -264,6 +264,10 @@ class _ProductPageState extends State<ProductPage>
             ),
           ),
         ),
+        margin: const EdgeInsets.symmetric(
+          horizontal: SMALL_SPACE,
```

Comment:
You're adding a horizontal spacing, aren't you?
And if you're not because the default behavior of `margin` includes a horizontal margin, that's a bit confusing to read: "hey, in this case there's a horizontal padding".

Reply:
Removed this and updated PR.